code/common/dqe.q

```diff
@@ -0,0 +1,6 @@
+/- checks if any variable for check function is type symbol
+.dqe.evaluate:{[table]
```

Comment:
```suggestion
\d .dqe

/- checks if any variable for check function is type symbol
evaluate:{[table]
```
This this file be in `code/dqcommon`? Currently it's loaded into every TorQ session as it is in `code/common`.

Reply:
@tszwailam do you have any idea about this? 

we can move it to `code/dqcommon` if you think it belongs there

Reply:
The file should be in common as it needed to be in every TorQ session. However this function is only used by .dqc.construckcheck to see if a table is in a HDB or not. 
I was thinking of changing .dqc.construckcheck , and just add a conditional in it, and remove the part where .dqe.evaluate is used.

If that change is ok, - I will remove the file completely as it will be useless